### DISCUSSION TITLE
Control AIO PUMP header fan speed

### DISCUSSION THREAD

**Discussion by Juancho-21** [Rem0o/FanControl.Releases#1399] [Q&A]
Hi, so I have a ITX Asus B450 I mobo with only one CHA_FAN, AIO_PUMP and CPU_CHAN.  I recently connect one regular fan to the AIO PUMP header (I can't connect more fans to the CHA_FAN header) and I tried everything to control the curve of RPM. sadly in the BIOS the AIO PUMP header is always at 100% and can't make any change to the curve...  with fan control i couldn't make the sensor appear and control it ... so im need some help! do I need to install some plug in ? or this header can't be controlled ? (PD with HWinfo im able to see the fan at full 100% speed all the time) 

Thanks

**Comment by bjdiss** [comment]
If the BIOS doesn't allow that fan to be controlled, I doubt any 3rd party software can do it either. It's not uncommon for an AIO pump to be locked at 100%.

**Comment by Juancho-21** [comment]
hi, so finally after some digging around the BIOS I could find to control the curve of the AIO (base on CPU usage) and still can't make Fancontrol to see it nor control it... do I need a plug in ? or am I configuring something wrong ?

**Reply by Rem0o** [reply]
You need to think of fan headers, not the hardware itself you plug in.  Fan Control sees headers.  You must plug your aio into a header fan control sees.

**Reply by Rem0o** [reply]
If you meant a header that is on the pump unit itself, FanControl won’t see it directly.

**Reply by Juancho-21** [reply]
hi thanks for the replay... I have 3 fans headers in the mobo: 
1.- CPU_FAN header: I have 1 fan with the cooler of the CPU. 
1.- CHA_FAN header: I have 3 daisy chain regular fans.
1.- AIO_PUMP header: here I connect 1 regular case fan and is the one I can't control 

I was able to control the AIO_PUMP header curve (it was always at 100% but digging in the BIOS I was finally able to modify the curve) but I want the curve to be configure to the GPU Temps not the CPU.... 

![IMAGE 2022-12-26 14:51:16](https://user-images.githubusercontent.com/121182403/209576499-0e763de1-f12e-4fdd-8e75-a94d2630ccb6.jpg)
![IMAGE 2022-12-26 14:51:14](https://user-images.githubusercontent.com/121182403/209576517-b05198db-fe7c-4b84-8feb-f32465ed338b.jpg)

**Reply by Rem0o** [reply]
Sorry to say but seems like your motherboard is badly "mapped" with the superIO chip it is recognized as. I don't have your motherboard to test, so I can't do much.

**Reply by Krops** [reply]
@Rem0o 
I have little bit another issue: I connected 3 fans to CPU fan header where two of them belongs to AIO and one regular PWN 4 pin fan.
Nzxt x53 AIO pump connected to AIO header. 3 pwn 4pin fans connected by one wire connected to chassis fan header. In FanControll I see following picture:
AIO pump detected, speed sensor works.
Fan1 (looks like 2 nzxt fans) can't be controlled but speed sensor works.
Fan2 regular PWN fan connected by one wire with AIO fans to CPU header, can be controlled and speed sensor works
Fan3, Fan4, Fan5 can't be controlled and speed sensors show 0 rpm for each fan
Fan6 - 3 fans connected to chassis fan header - can be controlled 3 fan together but no any available speed sensor for this
![Знімок екрана 2025-06-23 193109](https://github.com/user-attachments/assets/71a7cc61-1756-41e2-bf9e-02514c0d3eec)
![Знімок екрана 2025-06-23 193134](https://github.com/user-attachments/assets/e6cec1ca-aac6-46d2-a7dd-f57726f05e65)
I also have asus b450i where only cpu, aio and 1 chassis header. app version 227. I have suspisios that its some king of bios limitation because it recognized fan connected to cpu header as chassis fan and can't handle more than one chassis fan speed sensor
